__tests__/main.test.ts

```diff
@@ -0,0 +1 @@
+test('todo', async () => {});
```

Comment:
![image](https://user-images.githubusercontent.com/335103/108365397-75136c00-71f7-11eb-96d3-86cc9fce63de.png)

Reply:
I det mindste er der 1 test som test at ingenting virker

Reply:
I added tests for argument creation now